src/containers/markdown/MarkdownContainer.js

```diff
@@ -6,6 +6,7 @@ import Preview from '../../components/markdown/Preview';
 import styles from './MarkdownContainer.css';
 import { updateMarkdown } from '../../actions/markdownActions';
 
+
```

Comment:
why add this?

Reply:
We are unsure as to what you mean. In our file we are using both styles and updateMarkdown